lib/galaxy/tool_util/deps/mulled/mulled_build.py

```diff
@@ -57,7 +57,7 @@
 
 log = logging.getLogger(__name__)
 
-DIRNAME = os.path.dirname(__file__)
+DIRNAME = os.environ.get("INVFILE_DIRNAME", os.path.dirname(__file__))
```

Comment:
This seems fine, but maybe we can be more direct here ?
```suggestion
INVFILE = os.environ.get("INVFILE", os.path.join(os.path.dirname(__file__), "invfile.lua"))
```
and then change line 269 to use that variable directly ?

It might also be nice to add this to the argparse parse in add_build_arguments ?

Reply:
I've applied your suggestion!

But I'm afraid I don't follow you about `add_build_arguments` and `mull_targets`

Reply:
I've pushed a commit adding `--involucro-lua-file`, which defaults to INVFILE. That way users see it as a possible option when running `mulled-build --help`

Reply:
Awesome! Thanks!